frontend/javascripts/admin/admin_rest_api.js

```diff
@@ -648,8 +648,17 @@ export async function getTracingForAnnotationType(
   );
 
   const tracing = parseProtoTracing(tracingArrayBuffer, tracingType);
-  // The tracing id is not contained in the server tracing, but in the annotation content
+  // The tracing id is not contained in the server tracing, but in the annotation content.
   tracing.id = tracingId;
+  if (tracing.userBoundingBox != null) {
+    const oldBoundingBox = tracing.userBoundingBox;
+    const oldBoundingBoxId = Math.max(-1, ...tracing.userBoundingBoxes.map(bb => bb.id)) + 1;
```

Comment:
I think, we can use id `0` here, as it should be reserved or do you feel this is risky? Ideally the frontend would directly update the backend tracing, if it performs this conversion, so the old `userBoundingBox` can be deleted and the list is updated.

Reply:
0 is currently reserved for the first `userBoundingBox` and not directly dedicated to a possible old user bounding box. 
But if an old tracing with a bounding box is opened, it should be impossible to have the old user bbox while having a none empty array of userBoundingBoxes. Thus I think the default `0` should be fine.

Directly sending the updated tracing to the server isn't an easy thing on my first look. So I'll leave that out.